aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -0,0 +1,208 @@
+    | **Token Name**            | *&lt;token name&gt;*                                                                          | Enter a descriptive name for the token.                                                                |
+    | **Grant Type**            | Implicit                                                                                      |                                                                                                        |
+    | **Callback URL**          | `https://getpostman.com/postman`                                                              |                                                                                                        |
+    | **Auth URL**              | `https://login.microsoftonline.com/<tenant domain name>/oauth2/v2.0/authorize?p=B2C_1_SiUpIn` | Be sure to replace *&lt;tenant domain name&gt;* with the tenant's domain name with no angle brackets.  |
+    | **Client ID**             | *&lt;enter the Postman app's **Application ID**&gt;*                                          |                                                                                                        |
+    | **Client Secret**         | *&lt;leave blank&gt;*                                                                         |                                                                                                        |
+    | **Scope**                 | `https://<tenant domain name>/api/user_impersonation openid offline_access`                   | Be sure to replace *&lt;tenant domain name&gt;* with the tenant's domain name with no angle brackets.  |
+    | **Client Authentication** | Send client credentials in body                                                               |                                                                                                        |
+    
+3. Select the **Request Token** button.
+
+4. Postman opens a new window containing the Azure AD B2C tenant's sign in dialog. Sign in with an existing account (if one was created testing the policies) or select **Sign up now** to create a new account. The **Forgot your password?** link is used to reset a forgotten password.
+
+5. After successfully signing in, the window closes and the **MANAGE ACCESS TOKENS** dialog appears. Scroll down to the bottom and select the **Use Token** button.
+    
+    ![Where to find the "Use Token" button](./azure-ad-b2c-webapi/postman-access-token.png)
+
+### Test the web API with authentication
+
+Select the **Send** button to send the request again. This time, the response status is *200 Success* and the JSON payload is visible on the response **Body** tab.
+    
+![Payload and success status](./azure-ad-b2c-webapi/postman-success.png)
+
+## Next steps
+
+In this tutorial, you learned how to:
+
+> [!div class="checklist"]
+> * Create an Azure Active Directory B2C tenant
```

Comment:
Azure Active Directory B2C --> Azure AD B2C

Reply:
The list items get periods at the ends.